src/legend_data_monitor/plot_styles.py

```diff
@@ -227,6 +249,10 @@ def plot_scatter(
         # edgecolors=color,
     )
 
+    if plot_info["K_events"]:
```

Comment:
Is there a repetition? Already added at 226-229

Reply:
ok, nevermind, from the web interface I didn't notice the two pieces belong to two different functions. There is no repetition:)